internal/controller/platform/standalone/nimservice.go

```diff
@@ -121,6 +121,11 @@ func (r *NIMServiceReconciler) reconcileNIMService(ctx context.Context, nimServi
 		if err != nil {
 			return ctrl.Result{}, err
 		}
+	} else {
+		err = r.cleanupResource(ctx, &networkingv1.Ingress{}, namespacedName)
```

Comment:
Should the name of the ingress be set here. If all fields are empty it would not cleanup right?

Reply:
The name of ingress obj is sent across using the `namespacedName` variable

Reply:
I see. Missed the 'd' in the namespacedName variable. Thanks!